references/rpm_ostree_command_ref.adoc

```diff
@@ -0,0 +1,251 @@
+and create a new deployment with those packages installed.  Additionally, you can use
+the `rpm-ostree install` to layer local RPMs onto the system.  To remove layered
+packages, use `rpm-ostree uninstall`.
+
+By default, every `rpm-ostree` operation is "offline" - it has no effect
+on your running system, and will only take effect when you reboot.  This
+"pending" state is called the "pending deployment".  Operations can be chained;
+for example, if you invoke `rpm-ostree upgrade` after installing a package, your
+new root will upgraded with the package also installed.
+
+
+== Rebasing via `rpm-ostree rebase`
+
+Your operating system vendor may provide multiple base branches.  For example,
+Fedora Atomic Host has branches of the form:
+
+ - `fedora/26/x86_64/atomic-host`
+ - `fedora/26/x86_64/updates/atomic-host`
+ - `fedora/27/x86_64/atomic-host`
+
+You can use the `rebase` command to switch between these; this can represent a
+major version upgrade, or logically switching between different "testing"
+streams within the same release. Like every other `rpm-ostree` operation, All
+layered packages and local state will be carried across.
+
+The `rebase` command also allows you to switch your system to an entirely new
+OS distribution.  This means it is possible to switch your Fedora 26 Atomic Host
+to a version of CentOS Atomic Host.  While this is not likely to be done on a
+production system, it can be useful in certain testing scenrios.
+
+
+== Other local state changes
+
+=== `rpm-ostree cleanup`
+
+The `rpm-ostree cleanup` command allows the user to remove data generated by
+`rpm-ostree` operations which is no longer used.  This includes non-booted
+deployments, cached RPM repodata, partially downloaded RPMs, or transient
+allocated data as result of interrupted operations.
+
+The `-b` flag will remove the data remaining from interrupted operations.
+If you want to free up space on your system, use this options first.
+
+The `-m` flag will remove the cached RPM repodata and partially downloaded
+RPMs.
+
+The `-p` flag will remove the pending deployment (usually as a result of an
+upgrade or an install operation).
+
+The `-r` flag will remove the rollback deployment.
+
+You can combine all the flags in a single operation to cleanup all the data
+that can be cleaned up.
+
+
+=== `rpm-ostree initramfs`
+
+The `rpm-ostree intitramfs` command is primarily used to enable the regeneration
+of the initramfs after its configuration has changed or a kernel driver has been added.
+
+You can query the state of initramfs regeneration by using the command without any
+options.
+
+Or you can enable/disable the regeneration by using the `--enable` or `--disable`
+options respectively.
+
+=== `rpm-ostree reload`
+
+
+== Querying the RPM database
+
+=== `rpm-ostree db diff`
+
+=== `rpm-ostree db list`
+
+=== `rpm-ostree db version`
+
+
+== Composing an ostree commit
+
+=== `rpm-ostree compose tree`
+
+
+== Experimental interface
```

Comment:
Let's drop this section; the `ex livefs` and `ex override` are going to be covered elsewhere.  The other two are *really* experimental.